ETC.tsv

```diff
@@ -4765,7 +4765,7 @@ ETC_20150317_004764	{@st54}{b} Select Reward{/}
 ETC_20150317_004765	{memo X}Nudoku I was mounting the Son Bol pocket in Kochi.
 ETC_20150317_004766	Nudoku is available in Kochi!
 ETC_20150317_004767	Can be used in the tree Ferri!
-ETC_20150317_004768	$Magic charged (
+ETC_20150317_004768	$Magic Crystal Charged (
```

Comment:
Hmm no, here you're looking for the magic to use it to charge the crystal, so the crystal can be used to take out the vines.